modules/solace/solace.go

```diff
@@ -0,0 +1,125 @@
+package solace
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultVpn = "default"
+)
+
+type SolaceContainer struct {
+	testcontainers.Container
+	settings options
+}
+
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*SolaceContainer, error) {
+	// Default to the standard Solace image if none provided
+	settings := defaultOptions()
+	for _, opt := range opts {
+		if apply, ok := opt.(Option); ok {
+			if err := apply(&settings); err != nil {
+				return nil, err
+			}
+		}
+	}
+
+	req := testcontainers.ContainerRequest{
```

Comment:
suggestion: we are slowly moving to the pattern for directly use the Run function, instead of the ContainerRequest. See my last addition in the `dockermcpgateway` module [here](https://github.com/testcontainers/testcontainers-go/blob/main/modules/dockermcpgateway/dockermcpgateway.go#L76-L79). If you feel this can be done in a follow up, just let me know and I can do it afterwards. I'd prefer start establishing this pattern more and more from now on (I need to update the module generator 🤔 )